dv/uvm/core_ibex/scripts/collect_results.py

```diff
@@ -159,10 +159,14 @@ def main() -> int:
         for f in md.tests_pickle_files:
             try:
                 trr = TestRunResult.construct_from_pickle(f)
-                summary_dict[f"{trr.testname}.{trr.seed}"] = ('PASS' if trr.passed else 'FAILED')
+                summary_dict[f"{trr.testname}.{trr.seed}"] = \
+                    ('PASS' if trr.passed else
+                     'FAILED' + (" {T}" if (trr.failure_mode == Failure_Modes.TIMEOUT) else ""))
```

Comment:
How about changing T to TO?

Reply:
I'm just going to leave this as T to keep it as short as possible. Open to change if you feel strongly though.

Reply:
Maybe introduce one character abbreviations for all of the failure modes?

In addition the PASSED/FAILED stats could have a further failure mode breakdown.

Reply:
I think I will leave that as a future improvement for now. I suspect it may get crowded in the log. The machinery is there for us to report more interesting breakdowns.